src/Elmah.AzureTableStorage/AzureTableStorageErrorLog.cs

```diff
@@ -96,7 +93,8 @@ public override string Log(Error error)
                 StatusCode = error.StatusCode,
                 Type = error.Type,
                 User = error.User,
-            };
+            };
+            elmahEntity.SetXml(ErrorXml.EncodeString(error));
```

Comment:
How about using GZip only if the xml string exceeds 64kb ?
This is the only case it is mandatory to compress, otherwise it's a waste of computation.